plugin/vsnip.vim

```diff
@@ -56,6 +56,17 @@ function! s:open_command(bang, cmd)
   \ )))
 endfunction
 
+command! -range -bang -nargs=? VsnipAdd call s:add_command(<bang>0, 'vsplit', <line1>, <line2>, <q-args>)
+function! s:add_command(bang, cmd, start, end, name) abort
+  let lines = map(getbufline('%', a:start, a:end), { key, val -> json_encode(substitute(val, '\$', '\\$', 'ge')) })
+	let format = "  \"VsnipAdd\": {\n    \"prefix\": [\"%s\"],\n    \"body\": [%s]\n  }"
```

Comment:
I think `VsnipAdd` key should be empty text.

Reply:
Don't giving it a name results in vsnip not working.
I've changed it to the same as the prefix value.